docker/tester-exec.sh

```diff
@@ -15,6 +15,31 @@
 
 # This script relies on workspace which is cleaned via jenkins method
 
+# function to test and image with QEMU
+test_qemu() {
+  wget ${_WGET_OPTS} ${CI_BUILD_URL}/images/${MACHINE}/ovmf.qcow2
+
+  # Make port numbers and mac address that won't collide with other test runs
+  pid=$$
+  unique_port=$((pid%5000)) # Make a number between 0-4999
+  unique_mac=$((100000+(pid%900000))) # Make a number with 6 digits
+  PORT1=$((10000+unique_port))
+  PORT2=$((15000+unique_port))
+  QEMU_MAC=52:54:00:${unique_mac:0:2}:${unique_mac:2:2}:${unique_mac:4:2}
+
+  ./run-qemu.sh $PORT1 $PORT2 $QEMU_MAC $FILENAME &
+
+  QEMU_PID=$!
+  sleep 20 # Wait some time for QEMU to boot
```

Comment:
@mythi The reason for that sleep is that otherwise the expect script and the qemu will exit and the tests can't be run. There might be better solution but for now it will do.